_tools/redirects/redirects.csv

```diff
@@ -376,9 +376,9 @@ source,destination
 /tutorials/platform/android_in_app_purchases.html,/tutorials/platform/android/android_in_app_purchases.html
 /tutorials/plugins/android/android_plugin.html,/tutorials/platform/android/android_plugin.html
 /tutorials/plugins/android/index.html,/tutorials/platform/android/index.html
-/tutorials/plugins/gdnative/gdnative-c-example.html,/tutorials/scripting/gdnative/gdnative_c_example.html
-/tutorials/plugins/gdnative/gdnative-cpp-example.html,/tutorials/scripting/gdnative/gdnative_cpp_example.html
-/tutorials/plugins/gdnative/index.html,/tutorials/scripting/gdnative/index.html
+/tutorials/plugins/gdextension/gdextension-c-example.html,/tutorials/scripting/gdextension/gdextension_c_example.html
+/tutorials/plugins/gdextension/gdextension-cpp-example.html,/tutorials/scripting/gdextension/gdextension_cpp_example.html
+/tutorials/plugins/gdextension/index.html,/tutorials/scripting/gdextension/index.html
```

Comment:
This is still wrong, shouldn't be touched.

Reply:
This is odd. Should I remove these lines or the ones at the bottom of the file? They are essentially doing the same thing (if I would correct the ones you marked)

Reply:
They are not doing the same thing. These are old redirects which should be kept intact. You add new ones on top of them. The ones you've added are fine. Except this one:

```
/tutorials/scripting/gdnative/index.html,/tutorials/plugins/gdextension/index.html
```

It should be

```
/tutorials/scripting/gdnative/index.html,/tutorials/scripting/gdextension/index.html
```